### PR TITLE
fix wrong type assertion

### PR DESCRIPTION
**Pull request by mtcherni95**

### CODE REVIEW

File: tracee-ebpf/tracee/events_processor.go [hunk 1]
```diff
@@ -155,19 +156,19 @@ func (t *Tracee) processEvent(ctx *context, args map[string]interface{}, argMeta
...
-					if !ok || lastCtime != sourceFileCtime {
+					if !ok || lastCtime != castedSourceFileCtime {
 						//capture
 						err = CopyFileByPath(sourceFilePath, destinationFilePath)
 						if err != nil {
 							return err
 						}
 						//mark this file as captured
-						t.capturedFiles[capturedFileID] = sourceFileCtime
+						t.capturedFiles[capturedFileID] = castedSourceFileCtime
```
Comment (yanivagman) [inline tracee-ebpf/tracee/events_processor.go:171]: I think this is the only place that requires to cast the ctime, right?
If so, you can avoid the declaration and usage of castedSourceFileCtime in other places, and only cast here

Comment (mtcherni95) [reply tracee-ebpf/tracee/events_processor.go:171]: we use it also in line 164 - need to have the casted variable, no?

Comment (yanivagman) [reply tracee-ebpf/tracee/events_processor.go:171]: You can compare without a cast, knowing that timestamp will never be longer than 63 bits (thus you can compare the "signed" timestamp to unsigned)

Comment (mtcherni95) [reply tracee-ebpf/tracee/events_processor.go:171]: not sure you can actually "Invalid operation: lastCtime != sourceFileCtime (mismatched types int64 and uint64)"

**Pull request merged**